docs/get_started/quickstart.rst

```diff
@@ -8,32 +8,359 @@ Quick Start
+            x = F.max_pool2d(x, 2, 2)
+            x = x.view(-1, 800)
+            x = F.relu(self.fc1(x))
+            x = self.fc2(x)
+            return x
+
+:code:`PyTorchTaskRunner` inherits from :code:`nn.module`, so changing your deep learning model is as easy as modifying the network layers (i.e. :code:`self.conv1`, etc.) into the :code:`__init__` function, and then defining your :code:`forward` function. You'll notice that unlike PyTorch, the optimizer is also defined in this :code:`__init__` function. This is so the model AND optimizer weights can be distributed as part of the federation.  
+
+******************************************
+Defining your own train and validate tasks
+******************************************
+
+If you continue scrolling down in :code:`src/taskrunner.py`, you'll see two functions: :code:`train_` and :code:`validate_`. These are the primary tasks performed by the collaborators that have access to local data. 
+
+.. code-block:: python
+
+    def train_(self, train_dataloader: Iterator[Tuple[np.ndarray, np.ndarray]]) -> Metric:
+        """
+        Train single epoch.
+
+        Override this function in order to use custom training.
+
+        Args:
+            train_dataloader: Train dataset batch generator. Yields (samples, targets) tuples of
+            size = `self.data_loader.batch_size`.
+        Returns:
+            Metric: An object containing name and np.ndarray value.
+        """
+        losses = []
+        for data, target in train_dataloader:
+            data, target = data.to(self.device), target.to(self.device)
+            self.optimizer.zero_grad()
+            output = self(data)
+            loss = self.loss_fn(output, target)
+            loss.backward()
+            self.optimizer.step()
+            losses.append(loss.detach().cpu().numpy())
+        loss = np.mean(losses)
+        return Metric(name=self.loss_fn.__name__, value=np.array(loss))
+
+
+    def validate_(self, validation_dataloader: Iterator[Tuple[np.ndarray, np.ndarray]]) -> Metric:
+        """
+        Perform validation on PyTorch Model
+
+        Override this function for your own custom validation function
+
+        Args:
+            validation_dataloader: Validation dataset batch generator. Yields (samples, targets) tuples
+        Returns:
+            Metric: An object containing name and np.ndarray value
+        """
+
+        total_samples = 0
+        val_score = 0
+        with torch.no_grad():
+            for data, target in validation_dataloader:
+                samples = target.shape[0]
+                total_samples += samples
+                data, target = data.to(self.device), target.to(self.device, dtype=torch.int64)
+                output = self(data)
+                # get the index of the max log-probability
+                pred = output.argmax(dim=1)
+                val_score += pred.eq(target).sum().cpu().numpy()
+        
+        accuracy = val_score / total_samples

```

Comment:
Suggest adding return metric also as you make a mention specifically in the comment below
```suggestion
        accuracy = val_score / total_samples
        return Metric(name='accuracy', value=np.array(accuracy))
```

Reply:
I missed this. Great catch!